x-pack/plugins/endpoint/common/types.ts

```diff
@@ -45,6 +45,11 @@ export class EndpointAppConstants {
    **/
   static ALERT_LIST_DEFAULT_PAGE_SIZE = 10;
   static ALERT_LIST_DEFAULT_SORT = '@timestamp';
+
+  /**
+   * Metadata
+   */
+  static ENDPOINT_THRESHOLD_OFFLINE_MS = 5000;
```

Comment:
Are we expecting documents every 5 seconds @pzl?  Seems like we should have a larger buffer in case there are some communication issues between the elastic endpoint and elasticsearch.

Reply:
great Q. @nnamdifrankie @kevinlog this was as placeholder value. Do we know what the expected document frequency is? And, as Charlie mentioned, maybe a ~ +100% buffer (depending on what the scale is).